README.md

```diff
@@ -92,6 +92,32 @@ The release notes for each release are provided in the `release-notes.md` file t
 * Somatic Copy Number Variant (CNV) data are provided in a modified [SEG format](https://software.broadinstitute.org/software/igv/SEG) for each of the [applied software packages](https://alexslemonade.github.io/OpenPBTA-manuscript/#somatic-copy-number-variant-calling).
   * The CNVkit SEG file has an additional column `copy.num` to denote copy number of each segment, derived from the CNS file output of the algorithm described [here](https://cnvkit.readthedocs.io/en/stable/fileformats.html).
   * The ControlFreeC TSV file is a merge of `*_CNVs` files produced from the algorithm, and columns are described [here](http://boevalab.inf.ethz.ch/FREEC/tutorial.html#OUTPUT).
+  * NOTE: The _copy number_ annotated in the CNVkit SEG file is annotated with respect to ploidy 2, however, the _copy number_ annotated in the ControlFreeC TSV file is annotated with respect to inferred ploidy from the algorithm, which is recorded in the `pbta_histologies.tsv` file. See the table below for examples of possible interpretations.
```

Comment:
Based on your comment below, I'm wondering if this is closer to your meaning:

```suggestion
  * NOTE: The _copy number_ annotated in the CNVkit SEG file is annotated with respect to ploidy 2, however, the _status_ annotated in the ControlFreeC TSV file is annotated with respect to inferred ploidy from the algorithm, which is recorded in the `pbta_histologies.tsv` file. See the table below for examples of possible interpretations.
```

Because I _think_ your point is that copy number is the same  information in both outputs but copy number status needs to be taken in the light of the ploidy, which the `status` column in the ControlFreeC output captures:

```R
> head(controlfreec_df)
# A tibble: 6 x 10
  tumor         chr    start      end `copy number` status  genotype uncertainty WilcoxonRankSumTestPvalue KolmogorovSmirnovPvalue
  <chr>       <dbl>    <dbl>    <dbl>         <dbl> <chr>   <chr>          <dbl>                     <dbl>                   <dbl>
1 BS_KRYYDC2X     1     9348    10906             4 gain    -              -1                     0.0146                  0.0374  
2 BS_KRYYDC2X     1    61541   894292             2 neutral AA              1.26                  0.0766                  0.273   
3 BS_KRYYDC2X     1  2226382  2228719             4 gain    AABB           65.0                   0.00275                 0.00510 
4 BS_KRYYDC2X     1 28970231 28989706             2 neutral AA            100                     0.174                   0.122   
5 BS_KRYYDC2X     1 33254731 33259405             3 gain    AAA           100                     0.000206                0.000853
6 BS_KRYYDC2X     1 40955925 40958262             3 gain    -              -1                     0.00992                 0.0344  
```
`tumor_ploidy` for `BS_KRYYDC2X` is `2`.